src/index.tsx

```diff
@@ -166,24 +166,34 @@ const SortableList = <TTag extends keyof JSX.IntrinsicElements = typeof DEFAULT_
       lastTargetIndexRef.current = targetIndex
 
       const isMovingRight = sourceIndex < targetIndex
+      const draggedItemRects = itemsRect.current[sourceIndex]
+      const nextItemRects = itemsRect.current[isMovingRight ? sourceIndex + 1 : sourceIndex - 1]
+
+      if (nextItemRects === undefined) {
+        return
+      }
+
+      const spacingY = isMovingRight
+        ? (nextItemRects?.top ?? 0) - draggedItemRects.bottom
```

Comment:
Can I use `??` and `?.` or will it break IE support our something else?

Reply:
Yes, you can, this will be transpiled by TS.